suricata/update/main.py

```diff
@@ -1129,6 +1150,8 @@ def _main():
     
     update_parser.add_argument("--no-merge", action="store_true", default=False,
                                help="Do not merge the rules into a single file")
+    update_parser.add_argument("--check-versions", action="store_true",
+                               help="Check version of suricata.")
```

Comment:
This should be a sub-command instead.  It should be called from the command line like:
```
suricata-update check-versions
```
similar to update-sources, list-sources, etc.

But I also wonder if it should run as part of the default operation as well.  I can see `check-versions` subcommand useful on its own, to run without doing an update.  But maybe we should run this as part of the update as well.  Make it as a subcommand first, then worry about that though.